patrick/movies/src/components/index.js

```diff
@@ -0,0 +1,7 @@
+import Home from './Home';
```

Comment:
It's not wrong per se to have this `index.js` file in the components directory, but I also don't think it is necessary.